Source/WebKit/UIProcess/RemoteLayerTree/RemoteLayerTreeNode.h

```diff
@@ -131,6 +133,11 @@ class RemoteLayerTreeNode : public CanMakeWeakPtr<RemoteLayerTreeNode> {
         m_asyncContentsIdentifier = identifier;
     }
 
+#if ENABLE(THREADED_ANIMATION_RESOLUTION)
+    void setAcceleratedEffectsAndBaseValues(const WebCore::AcceleratedEffects&, const WebCore::AcceleratedEffectValues&, RemoteLayerTreeHost&);
```

Comment:
It often makes sense for setters to take r-value references if possible.

Reply:
In this case we don't move the values, just copy/clone them. We may revisit this later.